### DISCUSSION TITLE
Weekly Discussion

### DISCUSSION THREAD

**Discussion by deshpande-pavan** [strimzi/strimzi-kafka-operator#6644] [Q&A] [answered]
Dear Strimzi team,

I am a developer and have been using/working on strimzi-kafka-operator in my team for the last 2 months. I would like to join the community to know more and seek help if needed. I have heard that there is a weekly connect on strimzi. if so, could you please add me  to that weekly connect so that I can keep myself updated with the latest changes.

@scholzj - could you please help me out here? I would really appreciate the help. 

Please let me know if you need anything from my side.

Thanks in advance!

**Comment by ppatierno** [comment, marked as answer]
Hi @deshpande-pavan we are really glad to hear that you are using Strimzi :-)
Regarding the community meeting it's actually every two weeks and you can find all the information at this link:

https://github.com/strimzi/strimzi-kafka-operator#strimzi-community-meetings

There are the minutes, the recordings and the calendar you can subscribe to.

**Reply by deshpande-pavan** [reply]
Thanks for the quick reply. Really appreciate it.

**Reply by deshpande-pavan** [reply]
Hi @ppatierno, I wanted to join the live session where PRs and proposals are discussed. Could you please help me get the invite? As I also have started working on Strimzi-kafka-operator, I could raise a PR and discuss with the core team. Thanks!

**Reply by ppatierno** [reply]
@deshpande-pavan it's not about to be invited. The link I shared has the Google calendar to subscribe too but even the meeting minutes doc together with the Zoom link for joining.

**Reply by scholzj** [reply]
Just to add ... you of course also don' need to wait for the call with any PRs. You can also sync with us on Slack or in the discussions or simply open the PR and we can discuss the details in there.

**Reply by deshpande-pavan** [reply]
Okay, got it. Thank you for your response @ppatierno, @scholzj.
